modules/govuk/manifests/apps/smartanswers/enable_running_in_draft_mode.pp

```diff
@@ -0,0 +1,73 @@
+# == Class govuk::apps::smartanswers::enable_running_in_draft_mode
+#
+# Enables running smartanswers for serving draft content.
+#
+# Intended to be used only in development.
```

Comment:
I'm not sure this is quite the case? This will improve the publisher experience with the draft site right?

Reply:
Classic copy and paste error.

Reply:
@cbaines It turns out this comment is correct. This manifest does only run in the dev vm.